docs/tutorials/cpp.md

```diff
@@ -91,7 +132,7 @@ fn panic(_info: &core::panic::PanicInfo) -> ! {
 }
 ```
 
-This can then be compiled with `cargo +nightly -Z build-std=std,panic_abort -Z build-std-features=panic_immediate_abort` for a minimal build.
+This can then be compiled with `cargo +nightly --release -Z build-std=std,panic_abort -Z build-std-features=panic_immediate_abort` for a minimal build.
```

Comment:
Question: can `--release` go in this position? It's usually written `cargo build --release`

Reply:
heh the `build` command is missing.